LibTest/async/MultiStreamController/addErrorSync_A02_t01.dart

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion void addErrorSync( Object error, [ StackTrace? stackTrace ])
+/// Adds and delivers an error event.
+///
+/// Adds an error like [addError] and attempts to deliver it immediately.
+/// Delivery can be delayed if other previously added events are still pending
+/// delivery, if the subscription is paused, or if the subscription isn't
+/// listening yet.
```

Comment:
... where "the subsciprion isn't listening yet" includes the entire synchronous run of `onListen`.
So even without the earlier `add(1)`, the `addErrorSync` would still be enqueued.